java/IcebergDatastreamSink/README.md

```diff
@@ -0,0 +1,47 @@
+# Flink Iceberg Sink using DataStream API Examples
+
+* Flink version: 1.15.2
+* Flink API: DataStream API
+* Iceberg 1.3.1
+* Language: Java (11)
+
+This example demonstrate how to use
+[Flink Iceberg Sink Connector](https://iceberg.apache.org/docs/latest/flink-writes/) with the Glue Data Catalog
```

Comment:
Also, worth adding a note that the schema is only used at runtime, so it could be passed as part of the configuration or other ways (no need to deep dive in this example). The implication is important, as the same JAR could be used for multiple table

Reply:
Worth mentioning schema evolution is not supported. 
Also AVRO logical types (decimal, timestamp, date, time...) are not supported, unless you tested it